### ISSUE TITLE
Not releasing in bulb mode

### ISSUE THREAD

**Issue by shsmad** [coral/freemote#6]
Hi, I've found, that «Half down -> Fully down -> Half up -> Fully up» works only when shutter speed is fixed. When I set shutter speed to BULB on ILCE-7C (for example, to control timings programmatically), something weird is happening:

1. I send «Half down -> Fully down», shutter is pressed in camera
2. I wait for some time while image is well exposed
3. I send «Half up -> Fully up» supposing it should release the shutter

But the shutter isn't released, the camera doesn't respond at all to «Half up -> Fully up». I was able to release the shutter only with sending «Half down -> Fully down» **second** time (in manual focus mode one can only send just «Fully down» both times).

What's even stranger is that remote controls work the same way: if you press and hold the button in BULB mode, the shutter will not release after you release the button on RC, but it will release after you press it again. Tested on aodelan rmt-p1bta and JJC RMT-P1BT.

**Comment by Violin9906** [comment]
Yes, the BLE protocol works more like a Time shutter than a Bulb shutter. You need to send a press and up at both the beginning and end of exposure.
